src/gui/transferlistwidget.cpp

```diff
@@ -208,9 +210,14 @@ inline QModelIndex TransferListWidget::mapFromSource(const QModelIndex &index) c
     return nameFilterModel->mapFromSource(index);
 }
 
-void TransferListWidget::torrentDoubleClicked(const QModelIndex& index)
+void TransferListWidget::torrentDoubleClicked()
 {
-    BitTorrent::TorrentHandle *const torrent = listModel->torrentHandle(mapToSource(index));
+    const QModelIndexList selectedIndexes = selectionModel()->selectedRows();
+    if (selectedIndexes.size() != 1) return;
```

Comment:
Sure but as i said i copied these from `renameSelectedTorrent()`. I guess i should change them in that method too. I'll change them if sledgehammer says so.. don't wanna do and undo if he doesn't like it.